docs/source/conf.py

```diff
@@ -29,6 +29,12 @@
 _release = {}
 exec(compile(open('../../traitlets/_version.py').read(), '../../traitlets/_version.py', 'exec'),_release)
 
+# -- ENSURE: Build docs with current development version of traitlets.
+# REQUIRED-BY: autodoc (autoclass, etc.)
+HERE = os.path.dirname(__file__)
+TOPDIR = os.path.join(HERE, "..", "..")
+sys.path.insert(0, os.path.abspath(TOPDIR))
```

Comment:
Why is that ? It seem to currently work [on RTD](http://traitlets.readthedocs.org/en/stable/trait_types.html?highlight=caselessstrenum#traitlets.CaselessStrEnum)